actionlist/app/components/contentHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+} from 'react-native';
+import { Colors } from '../lib/commons';
+
+const style = StyleSheet.create({
+  headerText: {
+    fontSize: 10,
+    color: Colors.IUCrimson,
+    fontFamily: 'BentonSansBold, Arial, sans-serif',
+    margin: 10,
+  },
+});
+
+const ContentHeader = ({ title }) => (
+  <View>
+    <Text style={style.headerText}>{title}</Text>
```

Comment:
I tried to use children and lint threw an error saying I need to have a prop validation for it